xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/web/ExportAction.java

```diff
@@ -80,6 +80,15 @@
      */
     private static String PAGE_SEPARATOR = "&";
 
+    /**
+     * Define the different format supported by the export.
+     */
+    private enum EXPORT_FORMAT {
```

Comment:
I think we normally use camel case for the enum type name and upper case for the enum values.